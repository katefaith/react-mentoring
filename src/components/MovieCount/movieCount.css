.movieCount {
    margin: 25px 0;
}

.movieCount span {
    font-weight: 700;
}