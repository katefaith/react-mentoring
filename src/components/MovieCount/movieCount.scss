.movie-count {
    margin: 25px 0;
    
    span {
        font-weight: 700;
    }
}