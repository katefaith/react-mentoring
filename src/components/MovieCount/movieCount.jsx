import React from 'react';
import PropTypes from 'prop-types';

import './movieCount.scss';

export const MovieCount = ({ moviesCount }) => (
  <p className="movie-count">
    <span>{moviesCount}</span>
    {' '}
    movies found
  </p>
);

MovieCount.propTypes = {
  moviesCount: PropTypes.number.isRequired,
};
