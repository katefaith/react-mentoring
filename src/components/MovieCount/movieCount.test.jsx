import React from 'react';
import { render } from '@testing-library/react';
import { MovieCount } from './movieCount';

describe('MovieCount', () => {
  test('render  correctly', () => {
    const { container } = render(<MovieCount moviesCount={5} />);
    expect(container).toMatchSnapshot();
  });
});