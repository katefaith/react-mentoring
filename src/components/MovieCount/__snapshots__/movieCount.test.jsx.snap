// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCount render  correctly 1`] = `
<div>
  <p
    class="movie-count"
  >
    <span>
      5
    </span>
     
    movies found
  </p>
</div>
`;
